website/content/docs/enterprise/index.mdx

```diff
@@ -89,26 +90,27 @@ The Consul Enterprise features that are available depend on your license and the
 
 Available Enterprise features per Consul form and license include:
 
-| Feature                                                  | [HashiCorp Cloud Platform (HCP) Consul] | [Consul Enterprise] | Legacy Consul Enterprise (module-based)           |
```

Comment:
I reformatted the markdown tables to be easier to edit. The only content change is the LTS row (which I'll note with a comment on this PR).